src/components/notifications/notifications.service.ts

```diff
@@ -120,6 +124,11 @@ export function getChannelConfigs(): Promise<ResponseType> {
     return get(URL)
 }
 
+export function getWebhookAttributes(): Promise<WebhookAttributesResponseType> {
+    const URL = `${Routes.NOTIFIER}/variables`
+    return get(URL)
```

Comment:
we can directly use the value. no need to create variable

Reply:
done